lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -163,7 +183,7 @@ export class LoginPage {
 
     goToLoginPage() {
```

Comment:
methods should be changed to return the proper promise instead of returning "this"
for example clearUsername